Makefile

```diff
@@ -234,53 +266,6 @@ build-gpt-neox-deepspeed-gpu: build-gpu-cuda-113-base
 		-t $(NGC_REGISTRY)/$(GPU_GPT_NEOX_DEEPSPEED_ENVIRONMENT_NAME)-$(VERSION) \
 		.
 
-ifeq ($(NGC_PUBLISH),)
-define CPU_TF28_TAGS
--t $(DOCKERHUB_REGISTRY)/$(CPU_TF28_ENVIRONMENT_NAME)-$(SHORT_GIT_HASH) \
--t $(DOCKERHUB_REGISTRY)/$(CPU_TF28_ENVIRONMENT_NAME)-$(VERSION)
-endef
-else
-define CPU_TF28_TAGS
--t $(DOCKERHUB_REGISTRY)/$(CPU_TF28_ENVIRONMENT_NAME)-$(SHORT_GIT_HASH) \
--t $(DOCKERHUB_REGISTRY)/$(CPU_TF28_ENVIRONMENT_NAME)-$(VERSION) \
--t $(NGC_REGISTRY)/$(CPU_TF28_ENVIRONMENT_NAME)-$(SHORT_GIT_HASH) \
--t $(NGC_REGISTRY)/$(CPU_TF28_ENVIRONMENT_NAME)-$(VERSION)
-endef
-endif
-
-export CPU_TF28_ENVIRONMENT_NAME := $(CPU_PREFIX_38)tf-2.8$(CPU_SUFFIX)
-export GPU_TF28_ENVIRONMENT_NAME := $(CUDA_112_PREFIX)tf-2.8$(GPU_SUFFIX)
-
-.PHONY: build-tf28-cpu
-build-tf28-cpu: build-cpu-py-38-base
```

Comment:
I added this as part of the PR intentionally and I'll add this change to the description. Since I removed `CUDA_111` as well, I decided I might as well remove these too.